src/app.js

```diff
@@ -105,6 +106,50 @@ function renderDescription(containerNode, description) {
     if (dlBuilder) {
         containerNode.appendChild(dlBuilder.build());
     }
+
+    //removing two initial entries in array i.e. 'Grid position' and/or 'Clock region'
+    keys.shift();keys.shift();
+
+    const cont = document.getElementById("showTileDetails");
+    cont.innerText = "";
+    if(!keys.length){
+        cont.style.display = "none";
+    }
+    else{
+        cont.style.display = "block";
+    }
+    
+    for (let x of keys) {
+        let s = document.createElement("div");
+        s.innerText = x;
+        s.className = "indivTile";
+        cont.appendChild(s);
+        s.addEventListener('click', function details() {
+            let v = document.getElementById("item-info");
+
+            let details = document.createElement("div");
+
+            let head = document.createElement("h4");
+            head.innerText = `Inner Details of ${x}`;
+
+            let tilesSet = document.createElement("div");
+            tilesSet.className = "tiles";
+            for(let no = 1 ; no <= 3 ; no++){
+                let i = document.createElement("span");
+                i.innerText = `Component ${no}`;
+                tilesSet.appendChild(i);
+            }
+
+            details.appendChild(head);
+            details.appendChild(tilesSet);
+
+            v.appendChild(details);
+
+            // const myTimeout = setTimeout(function clearDetails() {
```

Comment:
please do not commit commented out code (either remove it, or add some information why this comment is here)